src/redis_index.c

```diff
@@ -95,7 +95,26 @@ int InvertedIndex_RegisterType(RedisModuleCtx *ctx) {
 * TODO: Add index name to it
 */
 RedisModuleString *fmtRedisTermKey(RedisSearchCtx *ctx, const char *term, size_t len) {
-  return RedisModule_CreateStringPrintf(ctx->redisCtx, TERM_KEY_FORMAT, ctx->spec->name, len, term);
+  char buf_s[1024] = {"ft:"};
```

Comment:
Me too, but I don't think we should do it in a function ostensibly designed to simply format a string. My other branches have this functionality.